<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Options</class>
 <widget class="QDialog" name="Options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Shortcuts</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Settings</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,2,3">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Page Iterator Mode: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="partialBox">
             <item>
              <property name="text">
               <string>Word</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Block</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Line</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Paragraph</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Symbol</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="2,2,3">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>OCR Engine Mode:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="engineMode">
             <item>
              <property name="text">
               <string>Tesseract Only</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>LSTM Only</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Tesseract LSTM Combined</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Default</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="2,2,3">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Page Segmentation Mode: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="pageSegMode">
             <item>
              <property name="text">
               <string>Auto OSD</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Auto</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Single Column</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Single Block Vert Text</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Single Block</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Single Line</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Single Word</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Single Char</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Circle Word</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Sparse Text</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,2,3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Tesseract Data Dir: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="dataDir"/>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="2,2,3">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Tesseract Data File:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="dataFile"/>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="2,2,3">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Text Removal Method:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="fillMethod">
             <item>
              <property name="text">
               <string>Inpainting Fill</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Neighboring Fill</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>166</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Keyboard Shortcuts</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="listWidget">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="modelColumn">
              <number>0</number>
             </property>
             <item>
              <property name="text">
               <string>Save Current Page (Ctrl + S)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Open New Page (Ctrl + O)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Undo Last Change (Ctrl + Z)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Redo Last Change (Ctrl + Shift + Z)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Zoom In (Ctrl + =)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Zoom Out (Ctrl + -)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Paste Image From Clipboard (Ctrl + V)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Find Word/Phrase (Ctrl + F)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Add Text Group From Selection (Ctrl + A)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Remove Text Group From Selection (Ctrl + R)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Group all selections into one highlight (Ctrl + G)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Delete a single highlight from the page (Ctrl + D)</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
